dask/dataframe/core.py

```diff
@@ -2037,13 +2042,17 @@ def to_timestamp(self, freq=None, how='start', axis=0):
         df.divisions = tuple(pd.Index(self.divisions).to_timestamp())
         return df
 
-    def quantile(self, q=0.5):
+    def quantile(self, q=0.5, method='default'):
         """ Approximate quantiles of Series
 
         q : list/array of floats, default 0.5 (50%)
             Iterable of numbers ranging from 0 to 1 for the desired quantiles
+
+        Note: this implementation will use t-digest is `method` is set to `tdigest` and the
```

Comment:
Should put this as a parameter docstring:

```python
"""
...
method : {'default', 'tdigest', 'dask'}:
    What method to use. By default will use dask's internal custom algorithm (``'dask'``).
    If set to ``'tdigest'`` will use tdigest for floats and ints and fallback to the ``'dask'``
    otherwise.
...
"""
```